.github/workflows/coverage.yml

```diff
@@ -8,14 +8,15 @@ on:
 
 jobs:
   type-checking:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
+    continue-on-error: true
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit